lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -216,11 +216,22 @@ namespace :foreman_tasks do
         end
       end
 
+      def self.count_duration(task)
```

Comment:
This is fine for the time being, but maybe having it as a method in the `Task` class would be better?